admin_submit_quest.py

```diff
@@ -33,10 +34,18 @@ class Quest(db.Model):
     test_outputs = db.Column(db.Text, nullable=True)
     xp = db.Column(db.Enum('30', '60', '100', name='xp_points'), nullable=False)
     type = db.Column(db.String(20), nullable=True)
+    is_active = db.Column(db.Boolean, default=True, nullable=True)
 
     def __repr__(self):
         return f"QuestID={self.quest_id}, Quest Name='{self.quest_name}', Language='{self.language}', Difficulty='{self.difficulty}', XP='{self.xp}'"
     
+
+class ReportedQuest(db.Model):
```

Comment:
that class and all other functionalities(like routes and functions) related to that implementation should be in a separate file

Reply:
The functionality related to reporting quests will not be in separate file (we talked about this). I will only move the class **"ReportedQuest"** in the file `edit_quest_form.py`

Reply:
Last commit: 0787b9f
- The **ReportedQuest** class is moved to the file `edit_quest_form.py`